src/domain/schemas/user-schemas.js

```diff
@@ -105,7 +105,7 @@ export const userDBSchema = joi.object({
   _id: idOrKeySchema.required(),
   passwordHash: joi.string().allow(null).required(),
   email: joi.string().case('lower').allow(null).required(),
-  roles: joi.array().required().items(joi.string()),
+  role: joi.string().required(),
```

Comment:
We also could restrict here to: `joi.string().valid(...Object.values(ROLE)).required()`

Reply:
Indeed, added!
Apropo to the validity of the previous `roles` array, there were a few of empty arrays in the Prod DB 😬